packages/edit-site/src/store/reducer.js

```diff
@@ -90,54 +85,6 @@ export function homeTemplateId( state, action ) {
 	return state;
 }
 
-/**
- * Reducer for information about the navigation panel, such as its active menu
- * and whether it should be opened or closed.
- *
- * Note: this reducer interacts with the inserter and list view panels reducers
- * to make sure that only one of the three panels is open at the same time.
- *
- * @param {Object} state  Current state.
- * @param {Object} action Dispatched action.
- */
-export function navigationPanel(
-	state = { menu: MENU_ROOT, isOpen: false },
-	action
-) {
-	switch ( action.type ) {
-		case 'SET_NAVIGATION_PANEL_ACTIVE_MENU':
```

Comment:
We can now remove the actions setNavigationPanelActiveMenu, openNavigationPanelToMenu etc as they don't do anything.